lib/src/network/tcp.rs

```diff
@@ -59,6 +58,7 @@ pub struct Client {
   request_id:     String,
   readiness:      Readiness,
```

Comment:
the `readiness` member is not needed anymore, since we use the one from `Pipe`

Reply:
Fixed